pyproject.toml

```diff
@@ -38,6 +38,9 @@ trino = {version="^0.314.0", optional=true}
 presto-python-client = {version="*", optional=true}
 clickhouse-driver = {version="*", optional=true}
 duckdb = {version="^0.6.0", optional=true}
+dbt-artifacts-parser = "^0.2.4"
+dbt-core = "^1.0.0"
+filelock = "^3.9.0"
```

Comment:
Why is filelock added here?

Reply:
For some reason, poetry was removing this package and it was breaking my poetry install locally

Reply:
it's a dependency for poetry itself so I'm not sure why it would be removed

Reply:
```
(datadiff)     poetry remove filelock
Updating dependencies
Resolving dependencies... (2.5s)

Writing lock file

Package operations: 0 installs, 0 updates, 1 removal

  • Removing filelock (3.9.0)
  
.
.
(datadiff)     poetry install
ModuleNotFoundError: No module named 'filelock'
```

Reply:
No longer seeing poetry remove this package automatically. I've reverting adding it